web/build.gradle.kts

```diff
@@ -25,5 +25,9 @@ dependencies {
 }
 
 
-kotlin.target.browser { }
-
+kotlin {
+    target {
+        useCommonJs()
```

Comment:
Why do we need this?

Reply:
I got error "When accessing module declarations from UMD, they must be marked by both @JsModule and @JsNonModule" when I added context stuff.....https://github.com/bdudelsack/kotlin-fullstack-example recommended adding this